CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ macro(torch_mlir_enable_werror)
   endif()
 endmacro()
 
+if(MSVC)
+  add_definitions(-D_USE_MATH_DEFINES)
+endif()
```

Comment:
> 2\. Add `USE_MATH_DEFINES` to resolve `torch-mlir\lib\Conversion\TorchOnnxToTorch\DefaultDomainGtoP.cpp(709): error C2065: 'M_LOG10E': undeclared identifier`

Thanks, I can reproduce that error locally on Windows without this patch. With the patch, I can build `torch-mlir-opt` again.

My team's downstream usage of torch-mlir (https://github.com/iree-org/iree/tree/main/compiler/plugins/input/Torch/torch-mlir) also still works with this since we use our own standalone CMake file. Sometimes these top level options in projects overwrite each other (or one project wants it but another doesn't).